Assets/MixedRealityToolkit.Tests/PlayModeTests/SlateTests.cs

```diff
@@ -118,6 +118,32 @@ public IEnumerator Prefab_TouchZoom()
             yield return h.Hide();
         }
 
+        /// <summary>
+        /// Test ggv zooming instantiated from prefab
+        /// </summary>
+        [UnityTest]
+        public IEnumerator Prefab_GGVZoom()
+        {
+            InstantiateFromPrefab(Vector3.forward);
+
+            PlayModeTestUtilities.SetHandSimulationMode(HandSimulationMode.Gestures);
+
+            TestHand h = new TestHand(Handedness.Right);
+            yield return h.Show(new Vector3(0.0f, 0.0f, 0.6f));
+
+            TestHand h2 = new TestHand(Handedness.Left);
+            yield return h2.Show(new Vector3(-0.1f, 0.0f, 0.6f));
+
+            yield return h.SetGesture(ArticulatedHandPose.GestureId.Pinch);
+            yield return h2.SetGesture(ArticulatedHandPose.GestureId.Pinch);
+
+            yield return h.Move(new Vector3(0.005f, 0.0f, 0.0f), 10);
+            yield return h2.Move(new Vector3(-0.005f, 0.0f, 0.0f), 10);
+
+            Assert.AreEqual(0.5, panZoom.CurrentScale, 0.1, "slate did not zoom in using two ggv hands");
+
+            yield return h.Hide();
```

Comment:
I think missing a call to both rightHand.Hide() and leftHand.Hide() - I think this only hides the right hand right now.